app/views/visns/_show_table_row.html.erb

```diff
@@ -1,10 +1,30 @@
 <% facilities.each do |facility| %>
+  <% path = facility.class.name === "ClinicalResourceHub" ? "/crh/" + facility.visn.number.to_s : va_facility_path(facility)  %>
+  <% if facility.class.name === "ClinicalResourceHub" %>
```

Comment:
We can create a variable call `is_crh` and set the value to `facility.class.name === "ClinicalResourceHub"`. That way we can just call the variable here and on line `13`

Reply:
agreed.  :)